lib/tag-finder.coffee

```diff
@@ -8,14 +8,14 @@ SelfClosingTags = require './self-closing-tags'
 module.exports =
 class TagFinder
   constructor: (@editor) ->
-    @tagPattern = /(<(\/?))([^\s>]+)([\s>]|$)/
-    @wordRegex = /[^>\r\n]*/
+    @tagPattern = /(<(\/?))([^/\s>]+)([^/>]*)(\/?)(>)/
```

Comment:
In my comment I originally said:
> In this regex the pattern `[^/>]*` is replaces by `((?:[^/>]|[^>](?!>))*`

this was wrong, I meant `(?:[^/>]|[^>](?!>))*` (without the first bracket). I just fixed that in the comment. If you copied that, this may have caused the problem...
If you get stuck at this problem I can try debugging it too, but I don't want to do any double work.